subtests/docker_cli/dockerinspect/dockerinspect.py

```diff
@@ -33,7 +33,7 @@ class inspect_base(SubSubtest):
 
     @staticmethod
     def verify_same_configs(subtest, source, comp, ignore_fields=None):
-        for i in range(len(comp)):
```

Comment:
C0200: 36,8: inspect_base.verify_same_configs: Consider using enumerate instead of iterating with range and len